### PR TITLE
Review and edits

### PR DESCRIPTION
**Pull request by SubTachyon**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/README.md [hunk 1]
```diff
@@ -30,16 +47,13 @@ for row in table.read_all():
     print(row)
 ```
 
-## Documentation
-
-Documentation is available [here](https://gooddata-sdk.readthedocs.io)
 
-## Bugs/Requests
+## Bugs & Requests
```
Comment (pcerny) [inline gooddata-sdk/README.md:51]: Should be the same in `README` files `gooddata-sdk`, `gooddata-pandas` and `gooddata-fdw`

**Review by pcerny — CHANGES_REQUESTED** [review]

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

File: gooddata-sdk/README.md (hunk 1 above)
Comment (SubTachyon) [reply gooddata-sdk/README.md:51]: fix'd

**Review by pcerny — APPROVED** [review]

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-python-sdk-gate-PR64

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-sonarqube-scan-zuul-gate/57/ : SUCCESS in 52s
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-fossa-scanning-zuul-el7/57/ : SUCCESS in 1m 41s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-python-sdk-merge-compliance-check/57/ : SUCCESS in 22s

**Pull request merged**